libnmstate/nm/__init__.py

```diff
@@ -36,6 +37,7 @@
 checkpoint
 connection
 device
+dns
```

Comment:
This change seems irrelevant to this PR. Can we remove it?

Reply:
metadata module needs it as it uses nm.dns.
I do not know how it worked without it. It is not related to my changes.. strange.